src/Tribe/Integrations/WPML/WPML.php

```diff
@@ -80,6 +84,40 @@ protected function hook_filters() {
 			$category_translation = Tribe__Events__Integrations__WPML__Category_Translation::instance();
 			add_filter( 'tribe_events_category_slug', array( $category_translation, 'filter_tribe_events_category_slug' ), 20, 2 );
 		}
+
+		//// WORK IN PROGRESS!!!
```

Comment:
:no_entry_sign: No space found before comment text; expected "// // WORK IN PROGRESS!!!" but found "//// WORK IN PROGRESS!!!"

<sub>hash: 9140ae0e08d2a0078d086452f1bc49c0</sub>